apps/web/app/(navigation)/EditBudget.tsx

```diff
@@ -22,20 +22,76 @@ import {
   DatePickerWrapperStyled,
   SeparatorStyled,
   ButtonWrapperStyled,
+  ErrorMessageWrapper,
 } from "./CreateNewBudget.styled";
-
+import { useSession } from "next-auth/react";
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+import { env } from "env.mjs";
 import { icons } from "./CreateNewBudget";
+import { useState } from "react";
+import { ErrorMessage } from "ui";
+
 interface EditBudgetProps {
   budget: BudgetFixed;
   onClose: () => void;
-}
+};
+
+interface editedBudgetBEProps {
+  name: string,
```

Comment:
Please change the letter to a capital letter
```suggestion

interface EditedBudgetBEProps {
  name: string,
```

Reply:
Changed, thanks